segmentstore/server/src/main/java/io/pravega/segmentstore/server/store/ServiceBuilder.java

```diff
@@ -126,8 +126,8 @@ private ServiceBuilder(ServiceBuilderConfig serviceBuilderConfig, ServiceConfig
         String instancePrefix = getInstanceIdPrefix(serviceConfig);
         this.coreExecutor = executorBuilder.apply(serviceConfig.getCoreThreadPoolSize(), instancePrefix + "core", Thread.NORM_PRIORITY);
         this.storageExecutor = executorBuilder.apply(serviceConfig.getStorageThreadPoolSize(), instancePrefix + "storage-io", Thread.NORM_PRIORITY);
-        this.lowPriorityExecutor =
-                executorBuilder.apply(10, instancePrefix + "low-priority-cleanup", 2);
+        this.lowPriorityExecutor = executorBuilder.apply(serviceConfig.getLowPriorityThreadPoolSize(),
+                instancePrefix + "low-priority-cleanup", 2);
```

Comment:
Why can't you use `Thread.MIN_PRIORITY`?

Reply:
Sure, done.